Makefile

```diff
@@ -7,5 +7,46 @@ integration:
 	behave --tags=$(INTEGRATIONS) tests -f progress2
 
 PYTHON_VERSION ?= 3.8
+docker-test: export PYTHON_VERSION := $(PYTHON_VERSION)
+docker-test: export DOCKER_TESTING ?= true
+docker-test: export TESTING_URL ?= https://github.com/algorand/algorand-sdk-testing.git
+docker-test: export TESTING_BRANCH ?= control-my-test-branches # revert to master before merge
 docker-test:
-	PYTHON_VERSION='$(PYTHON_VERSION)' ./run_integration.sh
+	./integration.sh
+
+docker-ps:
+	cd test-harness && docker-compose ps
+
+docker-pause:
+	cd test-harness && docker-compose pause
+
+docker-unpause:
+	cd test-harness && docker-compose unpause
+
+# these are utilized if run locally:
+LOCAL_TESTENV ?= .local-env
+prep-local-testenv:
+	python tests/prep_local_testenv.py $(LOCAL_TESTENV)
+
+show-local-testenv:
+	echo "contents of $(LOCAL_TESTENV)"
+	cat $(LOCAL_TESTENV) || echo "DNE"
+
+# configure the repos and branches that a local test will build off of:
+build-for-local-test: export TESTING_URL ?= https://github.com/algorand/algorand-sdk-testing.git
```

Comment:
@algoidurovic @algochoi - Do these changes make sense in addressing the branch control issues that you encountered last week?

Reply:
This is great - much easier to see when it's consolidated into one file. Will this override the up_env file in local tests?

Reply:
yes - locally, this overrides Testing SDK's pulled in`.up_env`, while `make docker-test` still uses `.up_env`. 

This may be a new source of confusion. Some other approaches that could be taken:
* instead of having `prep-local-testenv` create a new file `.local-env` which supplants `.up_env`, have it overwrite `.up_env` (this seems a bit hacky and even more confusing)
* dispense with `prep-local-testenv` along with the python script and most of the `Makefile` exports by making permanent `.local-env`

The advantage of the current approach is that one can see all the repo url's and branches in the `Makefile`. But there's no rule that says that this is how things need to be.

Reply:
I personally think the second bullet point is a cleaner solution than including exports in the Makefile. It also is more consistent with the way it is done in sandbox, which uses the `config.[network]` files. I'm curious what others think.